isb_web/main.py

```diff
@@ -253,6 +254,8 @@ async def thing_list_metadata(
 
 @app.get(f"/{THING_URL_PATH}/", response_model=ThingPage)
 def thing_list(
+    user_agent: typing.Optional[str] = Header("no_user_agent"),
+    x_forwarded_for: typing.Optional[str] = Header("no_ip"),
```

Comment:
I didn't spend too much time on this as I knew it would run differently behind nginx and I don't have that configured locally (yet), so I do expect this part to change a bit once I experiment there.